nomad/plan_apply.go

```diff
@@ -149,6 +149,8 @@ func (s *Server) applyPlan(plan *structs.Plan, result *structs.PlanResult, snap
 	for _, alloc := range req.Alloc {
 		if alloc.CreateTime == 0 {
 			alloc.CreateTime = now
+		} else {
+			alloc.ModifyTime = now
```

Comment:
This is unexpected, I must have messed something up rebasing b-nomad-0.7.1 yesterday, will fix.

Reply:
Fixed